src/doc/src/reference/specifying-dependencies.md

```diff
@@ -195,16 +197,27 @@ varies by where the repo is hosted; GitHub in particular exposes a reference to
 the most recent commit of every pull request as shown, but other git hosts often
 provide something equivalent, possibly under a different naming scheme.
 
+> **Note**: Neither the `git` key nor the `path` key changes the meaning of the
```

Comment:
Looks pretty good! Thank you for the perfect explanation.

Just wonder why insert this in the middle of the section, but not at the end?

Reply:
I guess I didn't want two notes in a row, and this seemed like an okay place, but I ended up putting two notes in a row on the path section anyway, so it probably makes sense to do that here too.

Reply:
Sounds good to me. Let's put it at the bottom.

Reply:
Moved!